ggcore/sdk_messages.py

```diff
@@ -1,27 +1,103 @@
 import dataclasses
+import typing
 from dataclasses import dataclass
 
-from ggcore.utils import RequestAuthType
+from ggcore.utils import HttpMethod
 
 
 @dataclass
+class SdkServiceResponse:
+    statusCode: int = None
```

Comment:
There are a couple places for this, but these should probably be `typing.Optional[x]`, e.g. `typing.Optional[int]` in these types of cases where they have a default value.